content/_includes/birds.macros.njk

```diff
@@ -101,3 +103,47 @@ params:
     <span {{ utility.attr_if('class', class) }}>{{ name | capitalize }}</span>
   {%- endif -%}
 {%- endmacro %}
+
+{% macro card(
+  bird,
+  birds=none,
+  level=3,
+  link=true,
+  bio=false,
+  class=''
+) %}
+  {% set bird = birds | authorPage(bird) if (birds and bird | typeCheck('string')) else bird %}
+  {% set id = ['bird', bird.data.bird] | join('-') | slug %}
+
+  <article aria-labelledby="{{ id }}" class="{{ ['h-card', class] | join(' ') }}">
+    {%- call embed.media_block(
+      media=embed.svg('faces/' + bird.data.bird),
+      name='bird',
+      media_attrs={'class': 'u-logo media-image'}
+    ) -%}
+      {%- set attrs = {
+        'class': 'item-title p-name',
+        'id': id
+      } -%}
+
+      {# This should be using the `h` shortcode #}
+      <h3 {{ utility.show_attrs(attrs) }}>
+        {{ utility.link_if(
+          content=bird.data.title | mdInline | safe,
+          url=bird.page.url if link else none,
+          class='title-link u-url'
+        ) -}}
+      </h3>
```

Comment:
@jgerigmeyer for some reason, I get an error when I try to use the `h` shortcode here. Thoughts?

Reply:
There seems to be an issue using the shortcode inside the `call` block. I avoided it by adding a `set` in https://github.com/oddbird/oddleventy/pull/548/commits/a7c8a2a141f647385601f40621a982966365a3bf